### DISCUSSION TITLE
User Manual

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/TMAP8#42] [Q&A]
Hello,

Is there a User Manual and where can I find it? It is a very simple, pheraps stupid question, but all i can find around is the User Manual for TMAP 7 and a similar resource would be fantastic

Thank you

**Comment by cticenhour** [comment]
Thank you for your interest! There is not a formal TMAP8 user manual (in PDF form) like TMAP7. Instead, we have a website available for TMAP8 component documentation, verification examples, and other information. Please visit https://mooseframework.inl.gov/TMAP8!
